docs/articles/documentation/guides/basic-guides/interact-with-the-page.md

```diff
@@ -0,0 +1,307 @@
+---
```

Comment:
See the comments I left (including the last file) and apply where applicable. Write me if you have any questions.